src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -76,13 +78,29 @@ public List<RawEntityTypeSummary> getEntityTypeSummary(Set<UUID> entityTypeIds)
 
     Condition publicEntityCondition = or(field(privateEntityField).isFalse(), field(privateEntityField).isNull());
     Condition entityTypeIdCondition = isEmpty(entityTypeIds) ? trueCondition() : field("id").in(entityTypeIds);
-    return jooqContext.select(idField, nameField)
+    return jooqContext
+      .select(idField, nameField)
       .from(table(TABLE_NAME))
       .where(entityTypeIdCondition.and(publicEntityCondition))
       .fetch()
-      .map(
-        row -> new RawEntityTypeSummary(row.get(idField), row.get(nameField))
-      );
+      .map(row -> new RawEntityTypeSummary(row.get(idField), row.get(nameField)));
+  }
+
+  public void replaceEntityTypeDefinitions(List<EntityType> entityTypes) throws JsonProcessingException {
```

Comment:
This is done by deleting all from the DB and re-inserting; I considered more complex update/diff logic, but for an operation that is relatively quick and infrequent it seemed like overkill.